alacritty/src/event.rs

```diff
@@ -532,7 +532,11 @@ impl<N: Notify + OnResize> Processor<N> {
 
                         processor.ctx.terminal.update_config(&config);
 
-                        if processor.ctx.config.font != config.font {
+                        // We're also reloading font on thickness change, since we update
+                        // glyph cache
```

Comment:
The phrasing here is a bit weird, the `also` suggests to me that there was a previous comment already talking about something that was reloaded, but there isn't.

```suggestion
                        // Reload font on thickness change, since we need to update glyph cache
```

Though generally it seems a bit strange to me, since we don't actually want to do half of the things in this method? It also seems a bit strange to me to reload the font just for this, it doesn't really feel right.

Reply:
We should clear loader, which requires to reloading both caches in the same time, doing them separately doesn't work well, due to the fact that you should update atlases.

Reply:
Updated as well, so resolved.